spec/codebase/codebase_spec.rb

```diff
@@ -23,14 +23,4 @@
 
     expect(invalid_engine_paths).to be_empty
   end
-
-  it 'does not contain respond_to blocks' do
```

Comment:
Just curious...For the web application, we should avoid using `respond_to` and `format.json`? 🤔

Reply:
No, it is fine to use :-) It is just that normally we recommend rendering a view not just some HTML string directly.

Reply:
@hoangnguyen92dn For API application, there should not be any `respond_to`

But since we added the health page, I had to remove it 🙏